src/app/services/trips.service.ts

```diff
@@ -110,10 +111,14 @@ export class TripsService {
   }
 
 
-  getTrip(tripIndex: string) {
+  getTrip(tripIndex: string): Trip {
     return this.trips.filter((trip) => {
```

Comment:
@zeus999 I commented on your commit as well, imo this is not the best way to do it, this sort of traverses the whole array, check example app with getSelectedBook method you'll know what I am talking about, if required change it.